### PR TITLE
Add Jest snapshot examples

### PR DESCRIPTION
**Pull request by dbjorge**
#### Description of changes

Adds sample tests with comments describing how to baseline with Jest snapshots.

#### Pull request checklist

- [x] If this PR addresses an existing issue, it is linked: Fixes #0000
- [x] `yarn test` passes in all affected samples
- [x] Added any applicable tests

### CODE REVIEW

File: typescript-selenium-webdriver/tests/index.test.ts [hunk 1]
```diff
@@ -7,6 +7,9 @@ import * as path from 'path';
 import { Builder, By, ThenableWebDriver, until } from 'selenium-webdriver';
 import * as chrome from 'selenium-webdriver/chrome';
 import { promisify } from 'util';
+import * as Axe from 'axe-core';
+
+const TEST_TIMEOUT_MS = 30000;
```
Comment (Shobhit1) [inline typescript-selenium-webdriver/tests/index.test.ts]: Same comment here to move these constants to a different file which can be common! I wont block a PR on this though.

Comment (dbjorge) [reply typescript-selenium-webdriver/tests/index.test.ts]: I would normally agree, but thought maybe keeping everything visible in context was more valuable in this particular case for sample purposes. Will consider moving this + some of the ChromeDriver setup into a separate setup file in a separate PR

File: typescript-selenium-webdriver/tests/index.test.ts [hunk 2]
```diff
@@ -54,23 +55,77 @@ describe('index.html', () => {
         await driver.wait(until.elementLocated(By.css('h1')));
     }, TEST_TIMEOUT_MS);
 
-    it('only contains known accessibility violations', async () => {
-        // Run an accessibility scan using axe-webdriverjs
-        const axeResults = await AxeBuilder(driver).analyze();
+    // This test case shows the most basic example: run a scan, fail the test if there are any failures.
+    // This is the way to go if you have no known/pre-existing violations you need to temporarily baseline.
+    it('has no accessibility violations in the h1 element', async () => {
+        const accessibilityScanResults = await AxeBuilder(driver)
```
Comment (smoralesd) [inline typescript-selenium-webdriver/tests/index.test.ts]: This object and it's usage is a little confusing: `AxeBuilder` -> make me think we're building and Axe instance... at the end of the chain I can see `.analyze()`... which, well... doesn't seem to build me an Axe. I guess it's ok to have a `AxeRunner` or and `A11YAnalyzer` class that happens to use fluent syntax and the builder pattern-like API to set it up though.

Comment (dbjorge) [reply typescript-selenium-webdriver/tests/index.test.ts]: Renamed to AxeWebdriverjs to improve clarify

**Pull request merged**